lib/services/access_checker_test.go

```diff
@@ -977,3 +977,65 @@ func TestAccessChecker_EnumerateMCPTools(t *testing.T) {
 		})
 	}
 }
+
+func TestIdentityCenterAccountAccessRequestMatcher(t *testing.T) {
+	const localCluster = "cluster"
+
+	tests := []struct {
+		info         *AccessInfo
+		name         string
+		resource     types.AppServerV3
+		assertAccess require.ErrorAssertionFunc
+	}{
+		{
```

Comment:
Might want to add a `KindKubernetesCluster` test case as well.

Reply:
Kubernetes test already covered here https://github.com/gravitational/teleport/blob/master/lib/services/access_checker_test.go#L33